examples/README.md

```diff
@@ -52,8 +52,8 @@ md_toc github examples/README.md | sed -e "s/\s-\s/ * /"
   * [hpc-gke.yaml](#hpc-gkeyaml--) ![community-badge] ![experimental-badge]
   * [ml-gke](#ml-gkeyaml--) ![community-badge] ![experimental-badge]
   * [storage-gke](#storage-gkeyaml--) ![community-badge] ![experimental-badge]
-  * [gke-a3-megagpu](#gke-a3-megagpuyaml--) ![community-badge] ![experimental-badge]
-  * [gke-a3-highgpu](#gke-a3-highgpuyaml--) ![community-badge] ![experimental-badge]
+  * [gke-a3-megagpu](#gke-a3-megagpuyaml--) ![experimental-badge]
```

Comment:
This was recently changed to [core] in https://github.com/GoogleCloudPlatform/cluster-toolkit/pull/3059/files

can you please rebase ? (maybe this PR is not required then ? )

Reply:
I'm closing this PR then.